### PR TITLE
create home-page

### PR DESCRIPTION
**Pull request by DariaLavrushko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,11 +25,13 @@
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
 		"react-i18next": "^13.3.1",
+		"react-inlinesvg": "^4.1.0",
```
Comment (Igor-Klein-1988) [inline package.json]: Why? Do you use it?

File: package.json [hunk 2]
```diff
@@ -25,11 +25,13 @@
...
 		"react-redux": "^8.0.1",
 		"react-router": "^6.18.0",
 		"react-router-dom": "^6.17.0",
 		"react-select": "^5.7.7",
 		"react-spinners": "^0.13.8",
+		"react-svg": "^16.1.29",
```
Comment (Igor-Klein-1988) [inline package.json]: same

File: public/locales/en.json [hunk 3]
```diff
@@ -1,4 +1,15 @@
 {
+	"home": {
+		"header": "Products",
+		"add_to_card": " + Add to cart",
+		"wishlist": "Wishlist"
+	},
+	"baner": {
+		"view_auctions": "View auctions",
+		"main_text_part1": "Find new auction items",
+		"main_text_part2": "for yourself!!",
```
Comment (Igor-Klein-1988) [inline public/locales/en.json:10]: it isn't "Auktionsgegenstände!!" or "аукционные лоты!!"

File: src/app/store.ts [hunk 4]
```diff
@@ -1,10 +1,12 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 import userProductsReducer from '../features/user/wishProducts/productsSlice';
+import productsReducer from '../features/products/productsSlice';
 // import counterReducer from '../features/counter/counterSlice';
```
Comment (Igor-Klein-1988) [inline src/app/store.ts:4]: delete it

File: public/locales/de.json [hunk 5]
```diff
@@ -1,4 +1,15 @@
 {
+	"home": {
+		"header": "Produkte",
+		"add_to_card": " + In den Korb",
+		"wishlist": "Wunschliste"
```
Comment (DenysBabenko29) [inline public/locales/de.json]: "wishlist": "Merkzettel",

File: src/features/products/Products.tsx [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+					</div>
+					<div className="tab_content">
+						<div id="featured" className="tab_pane active show">
+							<div className="product__section--inner">
+								<div className="row row-cols-xl-5 row-cols-lg-4 row-cols-md-3 row-cols-2 mb--n30">
+									{products.map((product) => (
+										<div key={product.id} className="col mb-30">
+											<div className="product__items">
+												<div className="product__items--thumbnail">
+													<a className="product__items--link" href="product-details.html">
```
Comment (Igor-Klein-1988) [inline src/features/products/Products.tsx]: use NavLink

File: src/features/products/Products.tsx [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+													<span className="product__items--content__subtitle">{product.title}</span>
+													<div className="product__items--content__title">
+														<a href="product-details.html">{product.category}</a>
+													</div>
+													<div className="product__items--price">
+														<span className="product_current__price">${product.price}</span>
+													</div>
+													<ul className="product__items--action d-flex">
+														<li className="product__items--action__list">
+															<a
```
Comment (Igor-Klein-1988) [inline src/features/products/Products.tsx]: use NavLink

File: src/features/products/Products.tsx [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+																href="cart.html"
+															>
+																<Basket />
+																<span className="product_add__to--cart__text">
+																	{t('add_to_card')}
+																</span>
+															</a>
+														</li>
+														<li className="product__items--action__list">
+															<a className="product__items--action__btn" href="wishlist.html">
```
Comment (Igor-Klein-1988) [inline src/features/products/Products.tsx]: use NavLink

File: src/features/products/Products.tsx [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+																<Heart />
+																<span className="visually-hidden">{t('wishlist')}</span>
+															</a>
+														</li>
+														<li className="product__items--action__list">
+															<a
```
Comment (Igor-Klein-1988) [inline src/features/products/Products.tsx]: use NavLink

File: src/features/products/productsSlice.ts [hunk 10]
```diff
@@ -0,0 +1,36 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import ProductsState from './types/ProductsState';
```
Comment (Igor-Klein-1988) [inline src/features/products/productsSlice.ts]: add empty line

File: src/index.scss [hunk 11]
```diff
@@ -42,6 +42,8 @@
 @import './assets/scss/elements/header.module';
 
 // @import "elements/header";
+@import "./assets/scss/elements/_baner";
```
Comment (Igor-Klein-1988) [inline src/index.scss]: @import "./assets/scss/elements/_baner`.scss`

File: src/features/products/baner/Baner.tsx [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+	return (
+		<div className="baner--items baner_home3__slider--bg">
+			<div className="container-fluid">
+				<div className="baner--items__inner">
+					<div className="row row-cols-1">
+						<div className="col">
+							<div className="baner__content">
+								<p className="baner__content--desc desc1 mb-15">{t('view_auctions')}</p>
+								<h2 className="baner__content--maintitle h1">
+									{' '}
```
Comment (DenysBabenko29) [inline src/features/products/baner/Baner.tsx]: need to?

**Review by DenysBabenko29 — CHANGES_REQUESTED** [review]

File: public/locales/en.json (hunk 3 above)
Comment (DariaLavrushko) [inline public/locales/en.json:10]: What is it?

File: src/features/products/baner/Baner.tsx (hunk 12 above)
Comment (DariaLavrushko) [reply src/features/products/baner/Baner.tsx]: Need empty line hire

**Review by Igor-Klein-1988 — APPROVED** [review]

**Pull request merged**